tutorials/vr/developing_for_oculus_quest.rst

```diff
@@ -0,0 +1,126 @@
+
+First you need to go through the steps on the :ref:`doc_exporting_for_android`
+page. This leads you through installing the toolset that Godot
+needs to export to Android devices.
+
+Next you need the Quest Plugin. You can download it from the Asset
+Library or manually download it from `here <https://github.com/GodotVR/godot-oculus-mobile-asset>`__.
+
+Setting Up Godot
+----------------
+
+To get started Open up Godot and create a new project
+
+.. image:: img/quest_new_project.png
+
+Make sure to chose the GLES2 renderer. Due to optimisations in the
+Quest's GPU this backend is far better suited for the Quest.
+
+Copy the addons folder from the Oculus Mobile asset into your Godot
+project. Your project tree should looks something like this:
+
+.. image:: img/quest_project_tree.png
+
+Now we can start building our main scene. To keep things simple
+we'll stick to the bare minimum.
+
+- We add an ARVROrigin node first. 
+- Then we add three child nodes to the origin node, one ARVRCamera and two ARVRControllers.
+- Assign controller id 1 to the first ARVRController and rename that to LeftHand.
+- Assign controller id 2 to the second ARVRController and rename that to RightHand.
+- Finally we'll add a MeshInstance as a child node to our first ARVRController and create a box shape, resize the box so each side is set to 0.1. Now duplicate the MeshInstance and move it to the second ARVRController node. These will stand in for our controllers, we're keeping things really simple.
+
+.. image:: img/quest_scene_tree.png
+
+Now add a script to the main node and add the following code:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    extends Spatial
+
+    onready var ovr_init_config = preload("res://addons/godot_ovrmobile/OvrInitConfig.gdns").new()
+    onready var ovr_performance = preload("res://addons/godot_ovrmobile/OvrPerformance.gdns").new()
+
+    var perform_runtime_config = false
+
+    func _ready():
+        var interface = ARVRServer.find_interface("OVRMobile")
+        if interface:
+            ovr_init_config.set_render_target_size_multiplier(1)
+
+            if interface.initialize():
+                get_viewport().arvr = true
+
+    func _process(_delta):
+        if !perform_runtime_config:
+            ovr_performance.set_clock_levels(1, 1)
+            ovr_performance.set_extra_latency_mode(1)
+            perform_runtime_config = true
+
+Before we can export our project to our Quest we need to do three
+more things.
+
+First go into the project settings for our project and make sure that
+our main scene is the scene we run. Godot does not ask you to set
+this on export.
+
+..image:: img/quest_project_settings.png
+
+Then go into the export menu and configure a new Android export. Now
+I'm assuming you followed the instructions in the "Exporting for
+Android" page I refered to at the start. If you didn't you'll have
+some red messages on this screen and I highly recommend you revisit
+that page.
```

Comment:
Assuming anything is not good idea in general. 

If you see any red messages on this screen it means you haven't followed the "Exporting for Android" guide. I highly recommend you revisit that page.